src/sentry/issues/endpoints/group_events.py

```diff
@@ -38,29 +50,60 @@ class GroupEventsError(Exception):
     pass
 
 
+@extend_schema(tags=["Events"])
 @region_silo_endpoint
 class GroupEventsEndpoint(GroupEndpoint, EnvironmentMixin):
     publish_status = {
-        "GET": ApiPublishStatus.UNKNOWN,
+        "GET": ApiPublishStatus.PUBLIC,
     }
     owner = ApiOwner.ISSUES
 
+    @extend_schema(
+        operation_id="List an Issue's Events",
+        parameters=[
+            GlobalParams.ORG_ID_OR_SLUG,
+            IssueParams.ISSUES_OR_GROUPS,
+            IssueParams.ISSUE_ID,
+            GlobalParams.START,
+            GlobalParams.END,
+            GlobalParams.STATS_PERIOD,
+            GlobalParams.ENVIRONMENT,
+            OpenApiParameter(
+                name="full",
+                type=OpenApiTypes.BOOL,
+                location=OpenApiParameter.QUERY,
+                description="If this is set to true, the event payload will include the full event body, including the stacktrace. Set to true to enable.",
```

Comment:
nit: probably don't need the second sentence? maybe reword to "Specify `true` to include the full event body, including the stacktrace, in the event payload." to match existing documentation of bool params